pemits-api/src/test/java/com/pemits/webcare/api/appointment/repository/AppointmentRepositoryTest.java

```diff
@@ -131,6 +131,95 @@ public void testSaveAllShouldReturnAllSavedAppointments() {
         assertThat(appointmentRepository.count(), equalTo(2L));
     }
 
+    @Test
+    @DatabaseSetup("/dataset/department/department-config.xml")
+    @DatabaseSetup({
+            "/dataset/user/users-of-type-doctor.xml",
+            "/dataset/user/users-of-type-patient.xml"
+    })
+    @DatabaseSetup("/dataset/doctor/doctor-config.xml")
+    @DatabaseSetup("/dataset/patient/patient-config.xml")
+    @DatabaseSetup("/dataset/appointment/appointment-config.xml")
+    public void testFindAppointmentByIdShouldReturnAppointment() {
+        final Optional<Appointment> appointment = appointmentRepository.findById(MOCK_APPOINTMENT1_ID);
+
+        assertThat(appointment.isPresent(), equalTo(true));
+    }
+
+    @Test
+    @DatabaseSetup("/dataset/department/department-config.xml")
+    @DatabaseSetup({
+            "/dataset/user/users-of-type-doctor.xml",
+            "/dataset/user/users-of-type-patient.xml"
+    })
+    @DatabaseSetup("/dataset/doctor/doctor-config.xml")
+    @DatabaseSetup("/dataset/patient/patient-config.xml")
+    @DatabaseSetup("/dataset/appointment/appointment-config.xml")
+    public void testFindAllShouldReturnNotEmptyList() {
+        final List<Appointment> appointments = appointmentRepository.findAll();
+        assertThat(appointments.size(), greaterThan(0));
+    }
+
+    @Test
+    @DatabaseSetup("/dataset/department/department-config.xml")
+    @DatabaseSetup({
+            "/dataset/user/users-of-type-doctor.xml",
+            "/dataset/user/users-of-type-patient.xml"
+    })
+    @DatabaseSetup("/dataset/doctor/doctor-config.xml")
+    @DatabaseSetup("/dataset/patient/patient-config.xml")
+    @DatabaseSetup("/dataset/appointment/appointment-config.xml")
+    public void testGetOneShouldReturnAppointment() {
+        final Appointment appointment = appointmentRepository.getOne(MOCK_APPOINTMENT1_ID);
+        assertThat(appointment.getAppointmentDate(), is(LocalDate.now()));
```

Comment:
Use other logic than this `LocalDate` attribute which is variable in nature. This test case passes today only.